### DISCUSSION TITLE
Tina Cloud Integration

### DISCUSSION THREAD

**Discussion by aguirrerjg** [danielcgilibert/blog-template#36] [General]
I need to know how to integrate Tina Cloud with this template. Could you help me?

**Comment by danielcgilibert** [comment]
Hello, in the documentation there is a video that explains how to deploy Tina Cloud. 

Doc :
https://tina.io/docs/tina-cloud/

Video : 
https://www.youtube.com/watch?v=r9vzL_8PEW8
